backend.rb

```diff
@@ -51,12 +55,16 @@ class Backend < Sinatra::Base
     authorize_payment_response = HTTParty.post(authorize_url, authorize_request_options)
 
     if (authorize_payment_response && authorize_payment_response.code == 201)
+      # The purchase was authorized (and has essentially been created as a
+      # resource available at the location specified in the authorization
+      # requests's response). Capture it.
```

Comment:
There needs to be a separate explanation about the meaning of authorizing an order and capturing it.